Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XLGppDataStore.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.ContractsLight;
+using BuildXL.Engine.Cache.KeyValueStores;
+using BuildXL.Execution.Analyzer;
+using BuildXL.Scheduler.Tracing;
+using BuildXL.Utilities;
+using Google.Protobuf;
+
+namespace BuildXL.Analyzers.Core.XLGPlusPlus
+{
+    public class XLGppDataStore
+    {
+        /// <summary>
+        /// Rocks DB Accessor for XLG++ data
+        /// </summary>
+        public KeyValueStoreAccessor Accessor { get; set; }
+
+        public XLGppDataStore()
+        {
+
+        }

```

Comment:
why do you need this ctor?

Reply:
See below comment about moving openDatastore into the constructor

---
In reply to: [306046328](https://github.com/microsoft/BuildXL/pull/636#discussion_r306046328) [](ancestors = 306046328)